jsfiles/index.js

```diff
@@ -0,0 +1,10 @@
+const menu = document.querySelector('.menu');
+const mobileMenu = document.querySelector('.mobile-menu');
+menu.addEventListener('click', () => {
+  mobileMenu.classList.remove('display-mobile');
+});
```

Comment:
- [ ] An error occurs when the hamburger button is clicked, and the mobile menu does not display. Please fix this as it is an essential requirement 🙏 
### Error
![image](https://user-images.githubusercontent.com/46839548/174800344-eea5a356-30be-4b1b-8814-c6ee9d3c7aae.png)